lib/glific/flows/router.ex

```diff
@@ -320,6 +320,7 @@ defmodule Glific.Flows.Router do
   end
 
   defp find_category(%{type: "random"} = router, _context, _msg) do
+    Glific.Metrics.increment("A/B Node")
```

Comment:
@dlobo I think we can put these events' name shortcodes instead of the actual event label. It will help us to manage or in case we want to resuse the event name somewhere else.

Reply:
@pankaj-ag I have made requested changes